packages/@storefront/query/test/unit/search-box.ts

```diff
@@ -29,25 +29,25 @@ suite('SearchBox', ({ expect, spy, stub, itShouldConsumeAlias, itShouldProvideAl
       });
 
       describe('onKeyDown()', () => {
-        it('should call event.preventDefault() if keyCode is up or down', () => {
+        it('should call event.preventDefault() if key is up or down', () => {
           const preventDefault = spy();
 
-          searchBox.state.onKeyDown(<any>{ keyCode: 38, preventDefault });
-          searchBox.state.onKeyDown(<any>{ keyCode: 40, preventDefault });
+          searchBox.state.onKeyDown(<any>{ key: 'ArrowUp', preventDefault });
```

Comment:
Import and use the `const`

Reply:
I was thinking about this, and my concern is that using properties of the `KEYS` table will ensure that the functions under test are internally consistent (ie. that they account for those values in particular), but not that the values themselves are correct.

For example: if we changed the value of `KEYS.UP` from 'ArrowUp' to `__FOO__`, the test would still pass even though the value itself is invalid.

What do you think?

Reply:
After some discussion with @jrmykolyn, I suggested that we still extract these values into constants, but keep the constants local to the file. These are standard browser values, so having them in two files should be fine.

Reply:
just import the const pls

Reply:
Updated test suite to import `KEYS` table from `core`. Resolving.